frontend/src/components/dashboard/DashboardBlocksBlock.jsx

```diff
@@ -1,91 +1,105 @@
 import Link from "next/link";
+import React, { Component } from "react";
 
 import { Row, Col, Card } from "react-bootstrap";
 
 import Timer from "../utils/Timer";
 
-export default props => (
-  <Col xs="6">
-    <Link href="/blocks/[hash]" as={`/blocks/${props.blockHash}`}>
-      <a className="dashboard-blocks-block-link">
-        <Card className="dashboard-blocks-block">
-          <Card.Title className="dashboard-blocks-block-title">
-            #{props.blockHeight}
-          </Card.Title>
-          <Card.Body className="dashboard-blocks-block-content">
-            <p className="dashboard-blocks-block-content-p">
-              <img src="/static/images/icon-m-transaction.svg" />
-              {props.transactionsCount}
-            </p>
-            <Row noGutters>
-              <Col md="7" xs="7">
+export default class extends Component {
+  render() {
+    const {
+      blockHash,
+      blockHeight,
+      blockTimestamp,
+      transactionsCount
+    } = this.props;
+    return (
+      <Col xs="6" md="12">
+        <Link href="/blocks/[hash]" as={`/blocks/${blockHash}`}>
+          <a className="dashboard-blocks-block-link">
+            <Card className="dashboard-blocks-block">
+              <Card.Title className="dashboard-blocks-block-title">
+                #{blockHeight}
+              </Card.Title>
+              <Col>
                 <span className="dashboard-blocks-block-content-p-footer">
-                  {props.blockHash.slice(0, 6)}...
+                  {blockHash.slice(0, 12)}...
```

Comment:
Will it fit on mobile?

Reply:
<img width="426" alt="image" src="https://user-images.githubusercontent.com/33007476/70081100-bac92200-15bc-11ea-9363-bc94920c8661.png">
I think it is fine

Reply:
I change it to this
<img width="352" alt="image" src="https://user-images.githubusercontent.com/33007476/70082864-295baf00-15c0-11ea-8210-3854e6d4b143.png">